tests/test_utils.py

```diff
@@ -380,18 +380,18 @@ def test_parse_query_string(self):
         result = uri.parse_query_string(query_string)
         assert result['a'] == decoded_url
         assert result['b'] == decoded_json
-        assert result['c'] == ['1', '2', '3']
+        assert result['c'] == '1,2,3'
```

Comment:
Doing a quick look at this file we are loosing tests for csv=True.
Have I missed them?

Reply:
We have very extensive parametrized tests for `parse_query_string()`, including both values of `csv`. But I can resurrect this specific one too so that we are not losing anything _per se_.

Reply:
it seems that `test_to_query_str_encoding` does mostly that, but it's a bit harder to follow

Reply:
I have now resurrected that old original non-parametrized test too, as well as added a new combo to it.